### DISCUSSION TITLE
hELLO 스킨 건의

### DISCUSSION THREAD

**Discussion by minikupa** [pronist/hELLO#91] [Ideas]
![화면 캡처 2020-09-27 154637](https://user-images.githubusercontent.com/19676242/94358109-8ab5b700-00d9-11eb-833c-4577ec60760e.jpg)
우선 그리드 내의 아이템들의 길이가 같아졌으면 좋을 것 같습니다. staggered grid 같은 느낌의 그리드가 아니라면 길이가 통일된 게 더 깔끔할 거라 생각합니다.
![화면 캡처 2020-09-27 155125](https://user-images.githubusercontent.com/19676242/94358169-0fa0d080-00da-11eb-960d-5f5aa35a8c9a.jpg)
또 사진에는 여백이 잘 들어가는데 유튜브 영상에는 여백이 안 들어가는 것 같습니다. 여기에도 사진처럼 적절한 여백이 들어가면 좋을 것 같습니다.
![화면 캡처 2020-09-27 155239](https://user-images.githubusercontent.com/19676242/94358187-2a734500-00da-11eb-90c6-457235e5cb10.jpg)
마지막 사진은 리스트(오리지널)인데 리스트가 화면 상단에 딱 붙어있게 나옵니다. 여기에도 적절한 여백이 있으면 좋을 것 같습니다.

**Comment by minikupa** [comment]
이전 글, 다음 글이 글을 몇 % 읽었을 때(상대적)가 아닌 몇 줄, 몇 픽셀 정도를(절대적) 스크롤 했을 때 팝업으로 뜨는 듯 합니다.  그래서 짧은 글들은 아예 안뜨거나 막바지에 가서 뜨지만 긴 글들은 초반 부분에 팝업이 뜹니다.

**Comment by pronist** [comment]
@minikupa 일단 다른 부분은 살펴보겠지만, 이전 글, 다음 글은 스크롤 기준이 아니라 글에 머무른 시간을 기준으로 한답니다 :)
즉 몇 초뒤에 나타나라. 이것이 전부인 것이죠.

**Comment by pronist** [comment]
@minikupa 반영결과를 알려드립니다.

* 그리드 - **그리드의 높이가 고정**되었습니다. **이미지의 출력형태가 일부 변경되어 고정된 세로 사이즈를 제공**합니다. 이미지가 없을 경우는 제외입니다. (반영)
*  iframe 여백 - **현재 스킨은 에디터에서 보이는 화면과 실제 포스트의 화면이 비슷하도록 유도**하고 있습니다. 확인해보니 유튜브 소스코드를 복사하여 영상을 가져오면 에디터 상에도 여백이 없는 것을 확인했습니다. 따라서 반영하지 않습니다. (반영x)
* 리스트(오리지널) 여백 - 이 부분은 **버그**였기 때문에 수정되었습니다. (반영)
 
기여해주셔서 감사합니다.
